content/blog/iva-fingerova-interview.md

```diff
@@ -0,0 +1,85 @@
+
+Becoming a UX designer was another huge decision leading me to this point - I got the field experience with “reactive UX” (altering UI that is already up and running) but no experience with starting from scratch. I jumped on a wagon that rode 1000 mph and learned on the go by doing. I got a first-row seat from which I saw how much needs to be thought of before the UX designer steps in the project. From the business, development, and marketing point of view, I feel I just grew into a digital product manager out of necessity thanks to my ability to connect the dots.
+
+**Your previous experience includes being a CEO, Product Manager, Operations Director, and more roles which are a crossroad of technology, leadership, and business. A common friend of ours pointed out that it’s a very unique experience in the Czech market, and we’re all very curious about what those experiences taught you? And what kind of skills did they require?**
+
+Haha, it sounds rather dashing, doesn’t it? But seriously, I was just hungry for knowledge, feeling a bit late for the party. I was 30 at the starting line while others were at least 5 years younger than me. So I pushed myself out of my comfort zones to find out what is in me, what do I excel at without much effort, and what is the biggest pain in the A… It was a rather brutal method but I guess it paid out pretty well. 
+
+I’ve always been the underdog - since the first day of elementary school perhaps even earlier. I guess that's why I have such an urge to get the underdogs to thrive, make them see that they are more than enough as they are, and show them that bullshitting never gets long-term results. 
+
+Thanks to all these opportunities I could define myself as a person that can work mainly with the growth mindset people in a transparent and maximum inclusive environment that supports individuals. And I’m so happy I found such a place!
+
+**Your current role has an impressive title - Head of Product Development at Invity.io, a company developing cryptocurrency rate comparison tool, where you’re currently building a new team. How’s that going? What’s most challenging in your current role? And would you mind telling us about the project you’re working on?**
+
+Well, Invity was just founded when I joined the team. At that time I was a bit shaken after my previous job and I was so glad to step down from leadership and slowly gather my strength. During those 15 months or so I channeled almost all my previous gigs. It was a rejuvenating process - I was a client, a PM, UX designer and UI designer (ehm), tester, researcher, and more, to make sure things were done in a “good enough” way. It was all necessary and I got to appreciate everything I’ve learned in my past and it helped to make invity.io a success. 
```

Comment:
At the end of this paragraph `invity.io` is written with small letter. I think it should be capitalized?